website/src/content/help-needed.mdx

```diff
@@ -46,3 +46,82 @@ This implies that changes to the [Custom Application configuration](https://docs
 If you are interested in more advanced functionalities, let us know and open a [support issue](https://github.com/commercetools/merchant-center-application-kit/issues/new/choose).
 
 </Info>
+
+# I want to use a test or staging environment for my Custom Application
+
+During the development cycle of new features for a Custom Application, it can be helpful to showcase or test those new features before publishing them to the production environment.
+
+This generally can be achieved using preview deployments, where you can inspect different versions of the Custom Application. As of now, the Merchant Center does not support this feature yet.
+
+The only option to emulate this setup is to create multiple Custom Applications in the Merchant Center, each one with a different [`entryPointUriPath`](/api-reference/application-config#entrypointuripath).
+
+However, the Custom Application must be configured to support "dynamic" values for the `entryPointUriPath` and build different versions of it.
+
+<Info>
+
+Custom Applications "dynamic" configurations (per environment) setup are further explained in [this section](/api-reference/application-config#environment-variable-references).
```

Comment:
I was wondering if something like this could also work:
```
Setting up dynamic configurations of Custom Applications (per environment) is further...
```